### ISSUE TITLE
Task payouts wiring

### ISSUE THREAD

**Issue by sprusr** [JoinColony/colonyDapp#943]
In the task details dialog, changing the payout token/value and submitting the form should change the value in the task DDB store (add an event to the store). And this value should be reflected when fetching the task from DDB.

- [x] Please rename 'bounty' to 'payout' if not already done
- [x] The `BOUNTY_SET` event should include information about the token for which the bounty/payout is (most likely just the token address)
- [x] The `setTaskBounty` command should include token information
- [x] The `getTask` query/reducer should include token information
- [x] A saga/action/reducer should be created which calls the `setTaskBounty` command and updates the value in the redux store
- [ ] The created action should be dispatched when submitting the task details dialog (appropriate payload blocked by #944)
- [ ] 🌮 the value should only be update in DDB if it has changed from the previous value (probably can be done in the saga, maybe command)
